angel_system/activity_hmm/core.py

```diff
@@ -154,8 +156,29 @@ def add_activity_classification(self, label_vec, conf_vec, start_time,
         self.times = np.hstack([self.times, times_])
         self.X = np.vstack([self.X, X])
 
+    def revert_to_step(self, step_ind):
```

Comment:
Is the "step" index here that of a time-step or the task-step index?

Reply:
The integer step as defined in the recipe.yaml via 'id'.